platform/view/services/comm/builder.go

```diff
@@ -160,7 +162,10 @@ func (p *P2PNode) startFinder() {
 }
 
 func (p *P2PNode) start() {
-	discovery.Advertise(context.Background(), p.finder, rendezVousString)
+	_, err := p.finder.Advertise(context.Background(), rendezVousString)
```

Comment:
Shall we pass the context to give the caller of the function to have a chance to interrupt the process? If the passed context is nil, then context.Background() is used.